packages/giselle-engine/src/core/generations/generate-image.ts

```diff
@@ -125,6 +126,7 @@ async function generateImageWithFal({
 		name: "ai-sdk/fal",
 		metadata: telemetry?.metadata,
 		input: { messages },
+		tags: ["deprecated"],
```

Comment:
bare-instrumentation on `generate-image.ts` is not deleted at this point because it holds cost for display
<img width="483" alt="image" src="https://github.com/user-attachments/assets/17c93353-0d3d-46e1-be92-8d04e1407ecf" />

it will be removed when https://github.com/giselles-ai/giselle/issues/796 is completed